app/assets/src/api/index.js

```diff
@@ -308,6 +308,11 @@ const createBackground = ({ description, name, sampleIds }) =>
     sample_ids: sampleIds,
   });
 
+const getBackgrounds = async () => {
+  const response = await get("/backgrounds.json");
```

Comment:
Be careful because this endpoint does not do access control.
We were supposed to add it as part of backgrounds/collections work but we have not started that project yet.